kcidb/__init__.py

```diff
@@ -252,11 +252,31 @@ def query_main():
 def schema_main():
     """Execute the kcidb-schema command-line tool"""
     sys.excepthook = misc.log_and_print_excepthook
-    description = 'kcidb-schema - Output latest I/O JSON schema'
+    description = 'kcidb-schema - Output latest or older I/O JSON schema'
     parser = misc.OutputArgumentParser(description=description)
+    schema_major_versions = []
+    version = io.schema.LATEST
+    while version:
+        schema_major_versions.append(version.major)
+        version = version.previous
```

Comment:
I see you're collecting the version numbers into `schema_major_versions` and then not using it. What was your intention here?